tests/test_resolver.py

```diff
@@ -13,8 +13,10 @@
     resolve_dependencies,
 )
 
+from .project_helpers import TarballPackage
+
 # The deps in each category should be disjoint
-other_deps = ["pandas", "numpy", "other"]
+other_deps = {"leftpadx": ["leftpad"]}
```

Comment:
>This replaces three deps with one, which removes some test coverage of the case where there is more than one "other" dep. I understand that these have to be available in our package cache in order for tests to work with install_deps == True, but could we add another package here like click or flit-core?

I didn't fully read your comment before implementing the solution so I went with another package than those you suggested. See [66b3d4f](https://github.com/tweag/FawltyDeps/pull/297/commits/66b3d4fbb8d67240a202b4b08996727baba0e234).

>Furthermore, the module added here does not "work" with the identity mapping (leftpadx vs leftpad). Is this change intenttional? Or does it not really matter to the tests?

The original `other_deps` object was a list when we were solely using identity mapping as the actual imports exposed by the package did not matter. Now that we're optionally using `install_deps`, `other_deps` need to define the imports exposed by each package so that those can be checked in the case `install_deps` is used. But those will be ignored in the case identity mapping is used.